metadata-ingestion/tests/integration/unity/unity_catalog_mces_golden.json

```diff
@@ -3813,6 +3826,52 @@
         "lastRunId": "no-run-id-provided"
     }
 },
+{
+    "entityType": "dataset",
+    "entityUrn": "urn:li:dataset:(urn:li:dataPlatform:databricks,dummy.acryl_metastore.quickstart_catalog.quickstart_schema.quickstart_table,PROD)",
+    "changeType": "UPSERT",
+    "aspectName": "datasetProfile",
+    "aspect": {
+        "json": {
+            "timestampMillis": 1703227191160,
+            "partitionSpec": {
+                "type": "FULL_TABLE",
```

Comment:
Should there be field-level stats here as well?

Reply:
yes, there will be but I did not add that in tests. I'll update tests to get that also.

Reply:
added field level profiling tests